DiscordBot/Core/CommandHandler.cs

```diff
@@ -33,7 +35,27 @@ public async Task StartCommandService()
             {
                 foreach(var command in commandGroups)
                 {
-                    Console.WriteLine("Loading Default Command for " + command.Name);
+                    Console.WriteLine("Creating command " + command.Name);
+                    Func<ICommandContext, object[], IServiceProvider, CommandInfo, Task> callback =
+                        (context, objects, service, commandInfo) =>
+                    {
+                        context.Channel.SendMessageAsync("Test");
+                        return Task.CompletedTask;
+                    };
+
+                    Action<CommandBuilder> commandBuilder = (a) =>
+                    {
+                        
+                    };
+
+                    Action<ModuleBuilder> newCommand = c => {
+                        c.AddCommand(command.Name, callback, commandBuilder);
+                    };
+                    
+                    await commands.CreateModuleAsync("test", newCommand);
```

Comment:
This code technically does work. But when you use `CreateModuleAsync` it requires you to tell the CommandHandler what module you want to run a command from. In this example to run the Default Initialized Command, you would have to run`>test {command.Name}`. Which is not viable because we want the user to be able to do `>{command.Name}` and have it give information about the command.

Reply:
Maybe inside of the CommandHandler script, we could have something that checks if the command is a part of the DefaultCommandInitializer.